### PR TITLE
wallet rpc should log queries

### LINKED ISSUE #561: wallet rpc should log queries
port https://github.com/bcoin-org/bcoin/pull/603/commits/7303d284d566eb609a41ecd378f137bfb1074b9a

### PR DESCRIPTION
**Pull request by nodar-chkuaselidze**
Closes #561 

I was thinking to add parameter logs as well, but we need to consider for each rpc command which should not be logged.
E.g. private key import parameters

And I think in general any "sensitive" information should not be logged in any mode.

I could go through RPCs and filter commands where we can log parameters, if you think it will be valuable :).

---

As mentioned in #561, this is port of:
https://github.com/bcoin-org/bcoin/commit/7303d284d566eb609a41ecd378f137bfb1074b9a

### CODE REVIEW

File: lib/node/rpc.js [hunk 1]
```diff
@@ -139,10 +139,9 @@ class RPC extends RPCBase {
         && cmd.method !== 'getblocktemplate'
         && cmd.method !== 'getbestblockhash') {
       this.logger.debug('Handling RPC call: %s.', cmd.method);
-      if (cmd.method !== 'submitblock'
-          && cmd.method !== 'getmemorypool') {
```
Comment (nodar-chkuaselidze) [inline lib/node/rpc.js:143]: `getmemorypool` is really old command that was replaced by `getrawmempool`.
I don't think bcoin ever supported it.

https://github.com/bitcoin/bitcoin/blob/1b6c463e033f861561d1a46ccf7eec069bbac09f/doc/release-notes/release-notes-0.7.0.md

**Comment by pinheadmz** [comment]
Awesome, can we add the logger context too? 

i.e. https://github.com/bcoin-org/bcoin/pull/603/commits/da2084f503e4012cfe673e66468a963a801ed538

Then we good. Thanks!

**Comment by nodar-chkuaselidze** [comment]
yep, good idea.
Done

**Review by pinheadmz — APPROVED** [review]
ACK d755331e685358a8bbcbbf4b10090df9a0907acd
<details><summary>Show Signature</summary>

```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

ACK d755331e685358a8bbcbbf4b10090df9a0907acd
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCAAdFiEE5hdzzW4BBA4vG9eM5+KYS2KJyToFAmBSOA0ACgkQ5+KYS2KJ
yTrkgg//c3iOrTKSak/vynDn9lAJGsFHSCMSS04+rB5vJqyHsFnEHHQhHfSX+L6r
msw1OtVZW587B6MMPXPyN2Og4ImgeC+yBXCtO/er51Zj99v7D6t8Dg1EzYLmGsmn
wdLDhTOuMWdhtak/g31KBmCMdsBXlnKsaCDAtKakhvhD5vMTYDTx6ak3yFTcjPbR
4s2a29MfNHhqLQDMagbwyH75mENdSJwamiDEE82KOxIYQGSgWZHnWIjxnPONSSX5
WDm5Nej4tKdYDy1uh3MiZNZRoXeAIGOWUcs1UpVizI66hH71bSjuZ/CDtg530SiT
XxiyN0xB34SH/TqlLKjNDZeIc8XE8G0n7fxXOeYzoDOBZll9ceEBn8yLJ7n1ECcz
AyQRzRyFdlqynNE0UyZscQpP5NZpCSwluP/oeaLBepJl5/PjwQsMPbHnSXh7awBq
jYubk22fVwqQNIbePp2nNWgG/J455HsSZ26pqwrptWENysZf6X5N1ygZjAxlZRwx
uH8XQNJZFjyBt2NyrF7h4480SI1btRI+yc0ixUPdfjQvimC7pdwtyeG/EZFtz0cw
frFYqH+U05go8Tf8P0mPUru780Vzete1WBt+76fDIKnNmLSyUddP60RViAwkj9jj
83BJzMibcYsbjztK6MpgfKR0+36dpCGnlerycb83SPe/fIEx0fI=
=QoHS
-----END PGP SIGNATURE-----
```

pinheadmz's public key is [on keybase](https://keybase.io/pinheadmz/pgp_keys.asc?fingerprint=e61773cd6e01040e2f1bd78ce7e2984b6289c93a)
</details>

**Pull request merged**